distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -799,18 +799,11 @@ public void testCswQueryWithValidationCheckerPlugin() throws Exception {
                 .all().assertThat().body(assertion[0], assertion);
         // @formatter:on
 
-        // Start metacard validation plugin; this will add on [validation-warnings = null] AND [validation-errors = null]
-        // filter to query
-        getServiceManager().startFeature(true, "catalog-plugin-metacard-validation");
```

Comment:
Does the feature run by default already or something?  

If not, wouldn't this change break the test?
Id so, any reason to have the assertion duplicated?